ModPatches/ABF Synstruct Biomimetics/Patches/ABF Synstruct Biomimetics/Bodies/Bodies_Animal_Quadrupedal.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<Operation Class="PatchOperationConditional">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Leg"]/groups</xpath>
+		<nomatch Class="PatchOperationAdd">
+			<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Leg"]</xpath>
+			<value>
+				<groups/>
+			</value>
+		</nomatch>
+	</Operation>
+
+	<Operation Class="PatchOperationConditional">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Leg"]/parts/li[def="ABF_BodyPart_Synstruct_Foot"]/groups</xpath>
+		<nomatch Class="PatchOperationAdd">
+			<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Leg"]/parts/li[def="ABF_BodyPart_Synstruct_Foot"]</xpath>
+			<value>
+				<groups/>
+			</value>
+		</nomatch>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Counterweight"]/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Neck"]/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Neck"]/parts/li[def="ABF_BodyPart_Synstruct_Head"]/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Neck"]/parts/li[def="ABF_BodyPart_Synstruct_Head"]/parts/li[def="ABF_BodyPart_Synstruct_AudialSensor"]/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Neck"]/parts/li[def="ABF_BodyPart_Synstruct_Head"]/parts/li[def="ABF_BodyPart_Synstruct_Jaw"]/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_QuadrupedalBiomimetic"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Leg"]</xpath>
+		<value>
+			<groups>
```

Comment:
duplicate `<groups>` field added by previous match operation, `/groups` to the xpath